src/main/java/seedu/address/logic/commands/AbstractFindCommand.java

```diff
@@ -42,6 +44,12 @@ protected ContainsKeywordsPredicate getPredicate() {
     public CommandResult execute(Model model) {
         requireNonNull(model);
         model.updateFilteredPersonList(this.predicate);
+
+        // if the result find list is empty
+        if (model.getFilteredPersonList().isEmpty()) {
+            return new CommandResult(String.format(MESSAGE_NO_PERSONS_FOUND));
```

Comment:
Is it necessary to have the `String.format` here? The `MESSAGE_NO_PERSONS_FOUND` itself should be enough to be taken as an argument of CommandResult constructor

Reply:
Resolved in commit 26c133fe3b549dc119e905ad649fed1c16a1c0fa